framer/GestureManager.coffee

```diff
@@ -207,11 +177,13 @@ splitStr = (str) ->
 
 addEventListeners = (target, types, handler) ->
 	splitStr(types).map (type) ->
+		#console.log("hammer.addEventListener", type)
 		Framer.CurrentContext.domEventManager.wrap(target)
```

Comment:
Mm.. I was using `getEventListeners` for this, but I just discovered it's only available in Chrome. And it only expects 1 node, not all of them. http://stackoverflow.com/a/16544813/617787